net/pfSense-pkg-WireGuard/files/usr/local/pkg/wireguard/wg.inc

```diff
@@ -0,0 +1,416 @@
+<?php
+/*
+ * wg.inc
+ *
+ * part of pfSense (https://www.pfsense.org)
+ * Copyright (c) 2021 Rubicon Communications, LLC (Netgate)
+ * Copyright (c) 2021 R. Christian McDonald
+ * Copyright (c) 2020 Ascord
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// pfSense includes
+require_once('config.inc');
+require_once('globals.inc');
+require_once('interfaces.inc');
+require_once('pfsense-utils.inc');
+require_once('pkg-utils.inc');
+require_once('service-utils.inc');
+require_once('system.inc');
+require_once('util.inc');
+
+// WireGuard includes
+require_once('wg_api.inc');
+require_once('wg_globals.inc');
+require_once('wg_install.inc');
+require_once('wg_validate.inc');

```

Comment:
All these files are installed under `/usr/local/pkg/wireguard`, but this directory is not part of PHP include_path variable.  You should prefix them with `wireguard/` because `/usr/local/pkg` is listed on include_path